### PR TITLE
typo cleanup

### PR DESCRIPTION
**Pull request by nschonni**
Ported some changes over from https://github.com/aspnet/EntityFramework.ApiDocs/pull/58
This can be broken up into smaller pieces if preferred

### CODE REVIEW

**Comment by ajcvickers** [comment]
@nschonni Thanks so much for all these PRs!

File: eng/common/SigningValidation.proj [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <!--
-    This MSBuild file is intended to be used as the body of the default 
+    This MSBuild file is intended to be used as the body of the default
```
Comment (dougbu) [inline eng/common/SigningValidation.proj]: Changes under eng/common are irrelevant. If you think this is important, make the changes in the dotnet/arcade repo.

File: src/EFCore.Relational/Migrations/MigrationsSqlGenerator.cs [hunk 2]
```diff
@@ -1505,7 +1505,7 @@ protected virtual IUpdateSqlGenerator SqlGenerator
         /// <param name="operation"> The operation. </param>
         /// <param name="model"> The target model which may be <c>null</c> if the operations exist without a model. </param>
         /// <param name="builder"> The command builder to use to add the SQL fragment. </param>
-        protected virtual void CreateTablePrimaryKeyContstraint(
+        protected virtual void CreateTablePrimaryKeyConstraint(
```
Comment (dougbu) [inline src/EFCore.Relational/Migrations/MigrationsSqlGenerator.cs]: Will AspNetCore need to react to this breaking change?

Comment (nschonni) [reply src/EFCore.Relational/Migrations/MigrationsSqlGenerator.cs]: It doesn't look like it's referenced there https://github.com/aspnet/AspNetCore/search?q=CreateTablePrimaryKeyContstraint&unscoped_q=CreateTablePrimaryKeyContstraint
And is a protected method

Comment (dougbu) [reply src/EFCore.Relational/Migrations/MigrationsSqlGenerator.cs]: Glad AspNetCore won't need a reaction.

This remains a breaking change; renaming a `protected` method breaks the public API unless the containing class is `internal` or `sealed`. For example, this PR would have needed updates to `ref` assemblies if it used them.

That said, I'm not sure this is announcement-worthy. @ajcvickers your thoughts?

**Pull request merged**